docs/getting-started-guides/ubuntu/automated.md

```diff
@@ -0,0 +1,253 @@
+and tag the issue with "juju" so we can find them.
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+- A working [Juju client](https://jujucharms.com/docs/2.0/getting-started-general); this does not have to be a Linux machine, it can also be Windows or OSX.
+- A [supported cloud](http://localhost:8090/#cloud-compatibility).
+
+### On Ubuntu
+
+On your local Ubuntu system:  
+
+```shell
+sudo add-apt-repository ppa:juju/stable
+sudo apt-get update
+sudo apt-get install juju
+```
+
+If you are using another distro/platform - please consult the
+[getting started guide](https://jujucharms.com/docs/2.0/getting-started-general)
+to install the Juju dependencies for your platform. 
+
+### Configure Juju to your favorite cloud provider
+
+Deployment of the cluster is [supported on a wide variety of public clouds](#cloud-compatibility), private OpenStack clouds, or raw bare metal clusters.  
+
+After deciding which cloud to deploy to, follow the [cloud setup page](https://jujucharms.com/docs/devel/getting-started-general#2.-choose-a-cloud) to configure deploying to that cloud. 
+
+Load your [cloud credentials](https://jujucharms.com/docs/2.0/credentials) for each 
+cloud provider you would like to use. 
+
+In this example
+
+```shell
+juju add-credential aws 
+credential name: my_credentials
+select auth-type [userpass, oauth, etc]: userpass
+enter username: jorge
+enter password: *******
+```
+
+You can also just auto load credentials for popular clouds with the `juju autoload-credentials` command, which will auto import your credentials from the default files and environment variables for each cloud. 
+
+Next we need to bootstrap a controller to manage the cluster. You need to define the cloud you want to bootstrap on, the region, and then any name for your controller node:   
+
+```shell
+juju bootstrap aws/us-east-2 example_cluster
+```
+or, another example, this time on Azure: 
+
+```shell
+juju bootstrap azure/centralus example_cluster_2
+```
+
+You will need a controller node for each cloud or region you are deploying to. See the [controller documentation](https://jujucharms.com/docs/2.0/controllers) for more information.
+
+Note that each controller can host multiple Kubernetes clusters in a given cloud or region. 
+
+## Launch a Kubernetes cluster
+
+The following command will deploy the intial 12-node starter cluster. The speed of execution is very dependent of the performance of the cloud you're deploying to, but 
+
+```shell
+juju deploy canonical-kubernetes
+```
+
+After this command executes we need to wait for the cloud to return back instances and for all the automated deployment tasks to execute. 
+
+## Monitor deployment
+
+The `juju status` command provides information about each unit in the cluster. We recommend using the `watch -c juju status --color` command to get a real-time view of the cluster as it deploys. When all the states are green and "Idle", the cluster is ready to go. 
+   
+
+```shell
+$ juju status
```

Comment:
This output is not canonical-kubernetes, and appears to be the ancient deployment of the hyperkube based charms.